### ISSUE TITLE
Install VILLASnode via CMake FetchContent

### ISSUE THREAD

**Issue by stv0g** [sogno-platform/dpsim#170]
Currently VILLASnode is installed via the Dockerfiles.

We should move this to CMake via a FetchContent module.

This might require that we install VILLASnode's dependencies also via FetchContent.

**Comment by stv0g** [comment]
@pipeacosta I think this is already related to your work?

**Comment by pipeacosta** [comment]
Yes, you can find the related CMake [here](https://github.com/pipeacosta/dpsim/commit/130befbdd793141f2b289e3b5b4f4fcc6a74737b), please let me know whether it is a good idea to prepare a pull request

**Comment by leonardocarreras** [comment]
Seem to be solved, also initial work done in #168
